curvefs/src/client/dentry_cache_manager.cpp

```diff
@@ -38,8 +46,10 @@ using NameLockGuard = ::curve::common::GenericNameLockGuard<Mutex>;
 
 void DentryCacheManagerImpl::InsertOrReplaceCache(const Dentry &dentry) {
     std::string key = GetDentryCacheKey(dentry.parentinodeid(), dentry.name());
-    NameLockGuard lock(nameLock_, key);
-    dCache_->Put(key, dentry);
+    if (!curvefs::client::common::FLAGS_enableCto) {
```

Comment:
In order to avoid adding FLAGS_enableCto everywhere, is it better to use a cto DentryCacheManagerImpl class